adobe_premiere_plugin/README.md

```diff
@@ -1,19 +1,91 @@
 # Adobe Premiere Plugin
 
-_Coming Soon_ – An Adobe Premier Plugin that brings the search power of
-Scrounger into Adobe Premiere is in development. The plugin will allow editors
-to access Scrounger natively in Premiere and import the results of their
-searches directly into their projects. Check the issues in the Github repo for
-updates on progress.
+An plugin that brings the search power of Scrounger into Adobe Premiere Pro.
+This allows editors to access Scrounger natively in Adobe Premiere Pro and
+import the results of their searches directly into their projects.
+
+## How It Works
+
+The panel is a simple static html page that lets the user fill in the location
+of their Scrounger application on the web, and the location to save the imported
+assets on their computer. Once the user clicks “Go”, they are redirected to the
```

Comment:
```suggestion
assets on their computer. Once the user clicks “Go,” they are redirected to the
```

Reply:
I don't understand this one 🤔

Reply:
it's just a grammar thing. commas go inside closing quotes.

Reply:
Thanks, I hate it ☹️